int8 motor_1
int8 motor_2
int8 motor_3
int8 motor_4
 